In [1]:
from ultralytics import YOLO

model = YOLO(f"yolo11n.yaml")
model = YOLO(f"yolo11n.pt")
results = model.train(data="trash.yaml", epochs=40,imgsz=1024,workers=16,batch=64,patience=5)
results = model.val()
  

New https://pypi.org/project/ultralytics/8.3.16 available 😃 Update with 'pip install -U ultralytics'
Ultralytics 8.3.15 🚀 Python-3.10.13 torch-1.12.1+cu116 CUDA:0 (Tesla V100-SXM2-32GB, 32501MiB)
WARNING ⚠️ Upgrade to torch>=2.0.0 for deterministic training.
engine/trainer: task=detect, mode=train, model=yolo11n.pt, data=trash.yaml, epochs=40, time=None, patience=5, batch=64, imgsz=1024, save=True, save_period=-1, cache=False, device=None, workers=16, project=None, name=train, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=No

wandb: Using wandb-core as the SDK backend. Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: jaejin. Use `wandb login --relogin` to force relogin


Freezing layer 'model.23.dfl.conv.weight'
AMP: running Automatic Mixed Precision (AMP) checks with YOLO11n...
AMP: checks passed ✅


FileNotFoundError: train: Error loading data from /data/ephemeral/home/level2-objectdetection-cv-20/code/dataset/yolo_train100
See https://docs.ultralytics.com/datasets for dataset formatting guidance.

In [ ]:
model = YOLO(f"yolo11m.yaml")
model = YOLO(f"yolo11m.pt")
results = model.train(data="trash.yaml", epochs=100,imgsz=1024,patience=5)
results = model.val()
  

Ultralytics 8.3.15 🚀 Python-3.10.13 torch-1.12.1+cu116 CUDA:0 (Tesla V100-SXM2-32GB, 32501MiB)
WARNING ⚠️ Upgrade to torch>=2.0.0 for deterministic training.
engine/trainer: task=detect, mode=train, model=yolo11m.pt, data=trash.yaml, epochs=100, time=None, patience=5, batch=16, imgsz=1024, save=True, save_period=-1, cache=False, device=None, workers=8, project=None, name=train3, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, 

Freezing layer 'model.23.dfl.conv.weight'
AMP: running Automatic Mixed Precision (AMP) checks with YOLO11n...
AMP: checks passed ✅


train: Scanning /data/ephemeral/home/level2-objectdetection-cv-20/dataset/yolo_train80/labels.cache... 4326 images, 0 backgrounds, 1 corrupt: 100%|██████████| 4327/4327 [00:00<?, ?it/s]

train: WARNING ⚠️ /data/ephemeral/home/level2-objectdetection-cv-20/dataset/yolo_train80/images/1148.jpg: ignoring corrupt image/label: image file is truncated (58 bytes not processed)



val: Scanning /data/ephemeral/home/level2-objectdetection-cv-20/dataset/yolo_val20/labels.cache... 1598 images, 0 backgrounds, 0 corrupt: 100%|██████████| 1598/1598 [00:00<?, ?it/s]


Plotting labels to runs/detect/train3/labels.jpg... 
optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.000714, momentum=0.9) with parameter groups 106 weight(decay=0.0), 113 weight(decay=0.0005), 112 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 1024 train, 1024 val
Using 8 dataloader workers
Logging results to runs/detect/train3
Starting training for 100 epochs...

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      1/100      22.1G      1.105      2.622      1.516         18       1024: 100%|██████████| 271/271 [02:09<00:00,  2.09it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:19<00:00,  2.59it/s]


                   all       1598      10886      0.223      0.161       0.11     0.0625

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      2/100      22.1G      1.285      2.395      1.657         23       1024: 100%|██████████| 271/271 [02:04<00:00,  2.18it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:44<00:00,  1.12it/s]


                   all       1598      10886      0.371      0.135      0.089     0.0489

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      3/100      22.1G      1.324      2.423      1.692         28       1024: 100%|██████████| 271/271 [02:02<00:00,  2.20it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:20<00:00,  2.39it/s]


                   all       1598      10886      0.197        0.2      0.138      0.083

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      4/100        22G      1.291      2.338      1.671         75       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.65it/s]


                   all       1598      10886      0.286      0.255      0.203      0.132

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      5/100      22.1G      1.238       2.27      1.631         35       1024: 100%|██████████| 271/271 [02:01<00:00,  2.22it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:21<00:00,  2.34it/s]


                   all       1598      10886      0.232      0.252      0.182      0.119

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      6/100      22.1G      1.187      2.154      1.591         45       1024: 100%|██████████| 271/271 [02:00<00:00,  2.26it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:20<00:00,  2.49it/s]


                   all       1598      10886      0.245      0.252      0.203      0.134

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      7/100      21.8G      1.172      2.138       1.58         53       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:19<00:00,  2.63it/s]


                   all       1598      10886      0.295      0.289      0.236      0.164

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      8/100        22G      1.146       2.06      1.558         65       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.66it/s]


                   all       1598      10886      0.331      0.313      0.276      0.189

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      9/100        22G      1.109      1.995      1.533         38       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:19<00:00,  2.62it/s]


                   all       1598      10886      0.365      0.334      0.289      0.202

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     10/100      22.1G      1.082      1.962      1.515         44       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:19<00:00,  2.62it/s]


                   all       1598      10886       0.41      0.308      0.311      0.219

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     11/100      21.9G      1.071      1.933      1.504         49       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:19<00:00,  2.62it/s]


                   all       1598      10886       0.46      0.356      0.345      0.249

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     12/100      22.1G      1.051      1.874      1.485         31       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.63it/s]


                   all       1598      10886      0.477      0.333      0.352      0.256

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     13/100        22G      1.024      1.839      1.467         62       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.65it/s]


                   all       1598      10886      0.459      0.342      0.344       0.25

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     14/100      22.1G      1.016      1.809      1.461         50       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886       0.46      0.378      0.374      0.277

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     15/100        22G      1.003      1.782      1.449         27       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.64it/s]


                   all       1598      10886       0.49      0.389      0.402      0.301

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     16/100      22.1G     0.9893      1.722      1.436         71       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.67it/s]


                   all       1598      10886      0.484       0.39      0.395      0.302

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     17/100      22.1G     0.9854      1.721      1.431        101       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.66it/s]


                   all       1598      10886       0.47      0.414        0.4      0.298

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     18/100      22.1G     0.9708      1.696      1.423         66       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.533      0.438      0.468      0.358

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     19/100        22G     0.9561      1.667       1.41         74       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.66it/s]


                   all       1598      10886      0.575       0.44      0.487      0.372

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     20/100        22G      0.954      1.644      1.412         61       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.67it/s]


                   all       1598      10886      0.534      0.438      0.465      0.351

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     21/100      22.1G     0.9423      1.626      1.403         76       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.548      0.425      0.457      0.354

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     22/100      22.1G     0.9285      1.589      1.391         61       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.66it/s]


                   all       1598      10886      0.566       0.44      0.486      0.383

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     23/100        22G      0.922      1.578      1.385         68       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.67it/s]


                   all       1598      10886      0.593      0.455      0.505      0.388

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     24/100      22.1G     0.9085      1.561       1.38         52       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.66it/s]


                   all       1598      10886      0.573      0.475      0.506      0.396

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     25/100      22.1G     0.9088      1.536      1.375         62       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.572      0.486      0.537      0.426

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     26/100      22.1G     0.9103      1.529      1.371         59       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.67it/s]


                   all       1598      10886      0.576      0.471      0.506      0.403

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     27/100        22G     0.8872      1.491      1.354         86       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.637        0.5      0.562      0.449

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     28/100        22G     0.8832      1.468      1.355         46       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.70it/s]


                   all       1598      10886      0.606      0.505      0.556      0.446

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     29/100        22G     0.8741      1.445      1.348         26       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.68it/s]


                   all       1598      10886      0.628      0.526      0.579      0.463

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     30/100      22.1G     0.8718      1.435      1.347        134       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.70it/s]


                   all       1598      10886      0.687      0.503      0.589      0.473

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     31/100        22G      0.864       1.42      1.338         34       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.71it/s]


                   all       1598      10886      0.656      0.534      0.594      0.478

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     32/100      22.2G     0.8471      1.365      1.323         54       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.68it/s]


                   all       1598      10886      0.665      0.544      0.615      0.499

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     33/100      22.1G     0.8481      1.377      1.331         95       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.674      0.536      0.614      0.499

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     34/100      22.1G     0.8395      1.363      1.322         56       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.67it/s]


                   all       1598      10886      0.689      0.535      0.609      0.492

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     35/100      22.1G     0.8402      1.355      1.324         98       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.68it/s]


                   all       1598      10886      0.702      0.555      0.638       0.52

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     36/100      22.1G     0.8292      1.311      1.311         58       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.723      0.555      0.643      0.527

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     37/100        22G     0.8191      1.297       1.31         99       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.71it/s]


                   all       1598      10886      0.688      0.592       0.66      0.541

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     38/100      22.1G     0.8163      1.299      1.304         71       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.697      0.568      0.652      0.536

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     39/100      22.1G     0.8143      1.286      1.295         85       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.719      0.577      0.661      0.543

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     40/100      22.1G     0.8018      1.264      1.294         36       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.70it/s]


                   all       1598      10886      0.727      0.603      0.677      0.559

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     41/100        22G      0.803      1.239      1.291         72       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.70it/s]


                   all       1598      10886      0.758      0.581       0.68       0.56

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     42/100      22.1G     0.7972      1.233      1.284         68       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.70it/s]


                   all       1598      10886       0.73      0.613      0.687      0.569

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     43/100      22.1G     0.7882      1.218       1.28         99       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886      0.743      0.618      0.701      0.588

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     44/100        22G     0.7797      1.194      1.271         52       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.71it/s]


                   all       1598      10886       0.75      0.623      0.709      0.592

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     45/100        22G     0.7906      1.174      1.274         62       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.68it/s]


                   all       1598      10886       0.76      0.628      0.722      0.607

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     46/100      22.1G     0.7759      1.168      1.268         61       1024: 100%|██████████| 271/271 [02:01<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.71it/s]

                   all       1598      10886      0.764      0.613       0.71      0.591



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     47/100      22.1G     0.7648      1.156      1.261         44       1024: 100%|██████████| 271/271 [02:01<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.71it/s]


                   all       1598      10886      0.759       0.63      0.719      0.601

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     48/100      22.1G     0.7613      1.125      1.255         46       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.71it/s]

                   all       1598      10886      0.778      0.653      0.746      0.623



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     49/100      22.1G      0.761      1.117      1.252         52       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.72it/s]

                   all       1598      10886      0.781       0.65      0.745      0.625



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     50/100      22.1G      0.749      1.102      1.248         42       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.69it/s]


                   all       1598      10886       0.79      0.658      0.752      0.635

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     51/100      22.1G     0.7502      1.076      1.245         30       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:19<00:00,  2.61it/s]

                   all       1598      10886      0.785      0.668      0.756      0.638



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     52/100      22.1G     0.7363      1.056      1.231         44       1024: 100%|██████████| 271/271 [02:13<00:00,  2.02it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:22<00:00,  2.24it/s]

                   all       1598      10886      0.788      0.661      0.755      0.639



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     53/100      22.1G     0.7319      1.046      1.226         54       1024: 100%|██████████| 271/271 [02:24<00:00,  1.87it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:22<00:00,  2.24it/s]


                   all       1598      10886      0.797      0.681      0.774      0.657

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     54/100      22.1G     0.7367      1.053      1.236         26       1024: 100%|██████████| 271/271 [02:02<00:00,  2.21it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.73it/s]


                   all       1598      10886      0.797      0.685      0.774      0.651

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     55/100        22G     0.7378      1.028      1.236         53       1024: 100%|██████████| 271/271 [02:00<00:00,  2.26it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.73it/s]

                   all       1598      10886      0.809      0.693      0.791      0.671



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     56/100      22.1G     0.7212      1.008      1.221         57       1024: 100%|██████████| 271/271 [02:09<00:00,  2.09it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:22<00:00,  2.26it/s]

                   all       1598      10886      0.818      0.711      0.801      0.684



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     57/100        22G       0.72     0.9924      1.218         76       1024: 100%|██████████| 271/271 [02:23<00:00,  1.89it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:22<00:00,  2.25it/s]

                   all       1598      10886      0.826      0.703        0.8      0.683



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     58/100      22.1G      0.715     0.9764      1.215         46       1024: 100%|██████████| 271/271 [02:09<00:00,  2.09it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:19<00:00,  2.61it/s]

                   all       1598      10886      0.815      0.715      0.804      0.687



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     59/100        22G     0.7087     0.9669       1.21         22       1024: 100%|██████████| 271/271 [02:01<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886      0.815      0.719       0.81      0.692



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     60/100      22.2G     0.6972     0.9367      1.202        109       1024: 100%|██████████| 271/271 [02:00<00:00,  2.24it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.74it/s]

                   all       1598      10886      0.852      0.716      0.825      0.707



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     61/100        22G     0.7038     0.9378      1.206         68       1024: 100%|██████████| 271/271 [02:13<00:00,  2.03it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886       0.84      0.735      0.826      0.709



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     62/100      22.1G      0.696      0.915        1.2         37       1024: 100%|██████████| 271/271 [02:09<00:00,  2.10it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:22<00:00,  2.27it/s]

                   all       1598      10886       0.85      0.742      0.832      0.717



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     63/100        22G     0.6965     0.9118        1.2         34       1024: 100%|██████████| 271/271 [02:23<00:00,  1.88it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:22<00:00,  2.27it/s]

                   all       1598      10886       0.86      0.734      0.836       0.72



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     64/100      22.1G     0.6972     0.9122      1.204         16       1024: 100%|██████████| 271/271 [02:21<00:00,  1.92it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:22<00:00,  2.20it/s]

                   all       1598      10886      0.853      0.746      0.842      0.726



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     65/100      22.1G     0.6827     0.8881      1.187         46       1024: 100%|██████████| 271/271 [02:23<00:00,  1.89it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.65it/s]

                   all       1598      10886      0.845      0.774       0.85      0.736



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     66/100      22.1G     0.6787     0.8769      1.185         28       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886      0.857      0.756      0.852      0.736



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     67/100      22.1G     0.6746     0.8584      1.182         25       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.74it/s]

                   all       1598      10886      0.856       0.77      0.853      0.742



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     68/100        22G     0.6675     0.8404      1.177         27       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:20<00:00,  2.43it/s]

                   all       1598      10886      0.875      0.772      0.864       0.75



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     69/100        22G     0.6644     0.8381      1.176         67       1024: 100%|██████████| 271/271 [02:14<00:00,  2.01it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.74it/s]

                   all       1598      10886      0.887      0.764      0.865      0.754



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     70/100      22.1G     0.6633     0.8334      1.173         53       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.76it/s]

                   all       1598      10886      0.867       0.78      0.867      0.757



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     71/100      22.1G     0.6583     0.8007      1.167         91       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886      0.889      0.775       0.87      0.761



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     72/100      21.8G     0.6545     0.7993      1.161         65       1024: 100%|██████████| 271/271 [02:00<00:00,  2.26it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886      0.885      0.786      0.877      0.766



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     73/100      22.1G     0.6487      0.785      1.161         33       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886      0.873      0.809      0.877       0.77



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     74/100        22G      0.643     0.7861      1.156         85       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.76it/s]

                   all       1598      10886      0.879      0.806      0.884      0.774



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     75/100        22G     0.6433     0.7647      1.154         93       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886      0.886       0.81      0.884      0.778



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     76/100      22.1G     0.6344     0.7521      1.148         43       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886      0.899      0.811      0.887      0.783



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     77/100      22.1G     0.6287     0.7456      1.147         50       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.76it/s]

                   all       1598      10886      0.895      0.816       0.89      0.784



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     78/100      22.1G     0.6208     0.7346      1.145         47       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.75it/s]

                   all       1598      10886      0.902      0.813      0.893      0.789



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     79/100        22G     0.6212     0.7262      1.139         56       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.77it/s]

                   all       1598      10886      0.898      0.814      0.896      0.795



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     80/100      22.1G     0.6221      0.723      1.144         46       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.74it/s]

                   all       1598      10886      0.908      0.823      0.901      0.797



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     81/100        22G      0.616     0.7151      1.134         28       1024: 100%|██████████| 271/271 [02:00<00:00,  2.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 50/50 [00:18<00:00,  2.76it/s]

                   all       1598      10886      0.918      0.812        0.9      0.798



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     82/100        22G      0.617     0.7226      1.137        142       1024:  37%|███▋      | 99/271 [00:44<01:16,  2.26it/s]

engine/trainer: 
task=detect, mode=train, model=yolo11m.pt, data=coco8.yaml, epochs=3, time=None, patience=100, batch=16, imgsz=640, save=True, save_period=-1, cache=False, device=None, workers=8, project=None, name=train, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, show_boxes=True, line_width=None, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=True, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, label_smoothing=0.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, bgr=0.0, mosaic=1.0, mixup=0.0, copy_paste=0.0, copy_paste_mode=flip, auto_augment=randaugment, erasing=0.4, crop_fraction=1.0, cfg=None, tracker=botsort.yaml, save_dir=runs/detect/train
